assemblies/pme/pom.xml

```diff
@@ -0,0 +1,160 @@
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>pentaho-karaf-features</groupId>
+      <artifactId>pentaho-big-data-plugin-osgi</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.activemq</groupId>
+      <artifactId>activemq-karaf</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.camel.karaf</groupId>
+      <artifactId>apache-camel</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pentaho-requirejs-osgi-manager</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho.webpackage</groupId>
+      <artifactId>pentaho-webpackage</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-dataservice</artifactId>
+      <classifier>features</classifier>
+      <type>xml</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <!-- Using maven-dependency-plugin instead of maven-assembly-plugin to copy files on an earlier
+       phase of the maven build pipeline -->
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>${maven-dependency-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>unpack</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.hitachivantara.karaf.assemblies</groupId>
+                  <artifactId>common-resources</artifactId>
+                  <version>${project.version}</version>
+                  <outputDirectory>target/classes</outputDirectory>
+                </artifactItem>
+              </artifactItems>
+              <excludes>/META-INF/**</excludes>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- karaf-maven-plugin will call both assembly and archive goals -->
+      <plugin>
+        <groupId>org.hitachivantara.karaf.tooling</groupId>
+        <artifactId>karaf-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-assembly</id>
+            <configuration>
+              <bootFeatures>
+                <feature>config</feature>
+                <feature>pentaho-client-minimal</feature>
+                <feature>pentaho-fasterxml</feature>
+                <feature>pentaho-dataservice-client</feature>
+                <feature>community-edition</feature>
+                <feature>package</feature>
```

Comment:
Is package real needed? Isn't already included on startup properties?

Reply:
package is necessary to run commands in ssh mode